modular_skyrat/modules/primitive_production/code/ceramics.dm

```diff
@@ -60,6 +60,7 @@
 	cost = CARGO_CRATE_VALUE * 2
 	unit_name = "ceramic product"
 	export_types = list(/obj/item/plate/ceramic,
+						/obj/item/plate/oven_tray/material/ceramic,
 						/obj/item/reagent_containers/cup/bowl/ceramic,
 						/obj/item/reagent_containers/cup/beaker/large/ceramic)
```

Comment:
```suggestion
	export_types = list(
		/obj/item/plate/ceramic,
		/obj/item/plate/oven_tray/material/ceramic,
		/obj/item/reagent_containers/cup/bowl/ceramic,
		/obj/item/reagent_containers/cup/beaker/large/ceramic,
	)
```

Reply:
I wanted these specific trays to have the value of ceramic items, since they require the same work to make.

Reply:
Nothing changed, it was just fixing the spacing.